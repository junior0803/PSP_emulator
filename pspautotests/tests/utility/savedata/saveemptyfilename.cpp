#include "emptyfilename.h"

extern "C" int main(int argc, char **argv) {
	initDisplay();

	runEmptyFileNameTrials("auto save", SCE_UTILITY_SAVEDATA_TYPE_AUTOSAVE, 0);
	runEmptyFileNameTrials("save:", SCE_UTILITY_SAVEDATA_TYPE_SAVE, 0);
	runEmptyFileNameTrials("list save:", SCE_UTILITY_SAVEDATA_TYPE_LISTSAVE, 0);
	runEmptyFileNameTrials("write data secure:", SCE_UTILITY_SAVEDATA_TYPE_WRITEDATASECURE, 0);
	runEmptyFileNameTrials("write data:", SCE_UTILITY_SAVEDATA_TYPE_WRITEDATA, 0);
	runEmptyFileNameTrials("make data secure:", SCE_UTILITY_SAVEDATA_TYPE_MAKEDATASECURE, 0);
	runEmptyFileNameTrials("make data:", SCE_UTILITY_SAVEDATA_TYPE_MAKEDATA, 0);

	return 0;
}