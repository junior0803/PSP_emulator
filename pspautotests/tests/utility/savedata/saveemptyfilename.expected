[x] auto save
[r] sceUtilitySavedataInitStart(mode:1): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[x] loaddata:Orignal data

[x] save:
[r] sceUtilitySavedataInitStart(mode:3): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[x] loaddata:Orignal data

[x] list save:
[r] sceUtilitySavedataInitStart(mode:5): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[x] loaddata:Orignal data

[x] write data secure:
[r] sceUtilitySavedataInitStart(mode:17): 00000000
[r] result: 00000000 => 80110327
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110327
[r] Files:
[x] System:
[x] loaddata:Orignal data

[x] write data:
[r] sceUtilitySavedataInitStart(mode:18): 00000000
[r] result: 00000000 => 80110327
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110327
[r] Files:
[x] System:
[x] loaddata:Orignal data

[x] make data secure:
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[x] loaddata:Orignal data

[x] make data:
[r] sceUtilitySavedataInitStart(mode:14): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[x] loaddata:Orignal data
